app/models/question.rb

```diff
@@ -0,0 +1,50 @@
+class Question < ApplicationRecord
+  include ::TokenHandler
+
+  belongs_to :user
+
+  before_create -> { generate_token(:published_token) }
```

Comment:
validate unique token

Reply:
already handled in generate token
discuss please